databases/week1/hyf_lesson1_db1hw.sql

```diff
@@ -0,0 +1,88 @@
+-- 1. Find out how many tasks are in the task table
+SELECT
+  COUNT(title)
```

Comment:
What if a task has a title that is NULL?

Reply:
it wont count in case if it is null , so as we discussed in the class it is best prcactise to use count(*) or count the primary key , right?

Reply:
Yes :)